Rubberduck.API/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Rubberduck.API")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("Rubberduck.API")]
+[assembly: AssemblyCopyright("Copyright ©  2018")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+// Setting ComVisible to false makes the types in this assembly not visible
+// to COM components.  If you need to access a type in this assembly from
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+// The following GUID is for the ID of the typelib if this project is exposed to COM
+[assembly: Guid("ac1b4a57-364a-4f90-a0cd-6ee818349ce5")]
```

Comment:
Aren't we supposed to pull it from the mega list of GUIDs somewhere?

Reply:
I can move it into that mega list if tha thelps.. we do only have such a list for CLSIDs and PROGIDs

Reply:
on second thought: for consistency we should also do that for "leaf" assemblies, but we can't really do that, because there's no one assembly that all others depend on...

Reply:
I'm pretty sure this is only needed for assembly that's going to be COM-visible because the assembly GUID is then used as the `TypeLib`'s GUID. If the assembly doesn't contain any COM types, then we need not worry about maintaining their GUIDs.

Reply:
Maybe we should consider creating a "utilities" project that all the other assemblies can use.